Makefile

```diff
@@ -2,6 +2,11 @@ deps:
 	pip install -e .[dev]
 .PHONY: deps
 
-test:
+lint:
+	flake8 --verbose dcos_cli/ tests/
+	pylint --disable=missing-docstring,cyclic-import dcos_cli/ tests/
```

Comment:
`pylint` configuration inevitably becomes long. I recommend using a pylint configuration file, but that can wait 😄